src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/EkspanderbartBegrunnelsePanel.tsx

```diff
@@ -43,8 +45,16 @@ const slutterSenereEnnInneværendeMåned = (tom?: string) =>
         endOfMonth(dagensDato)
     );
 
+const finnPresentertTomDato = (periodeFom?: string, periodeTom?: string, sisteFom?: string) => {
+    if (erSammeFom(periodeFom, sisteFom)) {
```

Comment:
Kunne man ha sjekket på id i stedet? Da kunne man ha sendt inn `sisteVedtaksperiodeId` og ha sjekket `vedtaksperiodeMedBegrunnelser.id === sisteVedtaksperiodeId`. Eller kan det være en annen periode, men som har samme id kanskje?

Reply:
Det vi prøver her er å se om det eksisterer en senere fom i vedtaket enn fom for perioden vi ser på. I så fall ønsker vi alltid å vise periodens tom. Hvis det ikke eksisterer en senere fom, skjuler vi tom hvis den slutter senere enn inneværende måned